content/docs/2.8/concepts/scaling-deployments.md

```diff
@@ -226,6 +226,22 @@ metadata:
 
 The presensce of this annotation will pause autoscaling no matter what number of replicas is provided. The above annotation will scale your current workload to 0 replicas and pause autoscaling. You can set the value of replicas for an object to be paused at to any arbitary number. To enable autoscaling again, simply remove the annotation from the `ScaledObject`definition.
 
+### Activating and Scaling thresholds
+
+There are user cases where the activating value (0-1 and 1-0) is totally different than 0, ie: workloads scaled with prometheus scaler where the values go from -X to X. KEDA brings support to specify different values for each case:
```

Comment:
Before we dive in to this, I would conceptually explain activation vs scaling, or point to our existing docs on this

Reply:
Isn't enough the explanation below? I can expand it

Reply:
No because it only comes later on, so people who are new to this might have no idea what we are talking about

Reply:
Agree with Tom on this one. We should start with explaining what is Activation and what is Scaling, then we can follow up with a different options.